### DISCUSSION TITLE
Pretrained

### DISCUSSION THREAD

**Discussion by felixVogel02** [mlfoundations/open_clip#103] [Q&A]
Hi, are the models f.e. ViT-B/32 only trained on LAION400M or are they also pretrained on other datasets? And if they are pretrained on other datasets, on which datasets are they pretrained?

Thanks for your great work!

Felix

**Comment by mitchellnw** [comment]
Hello,

This is specified with the `pretrained` argument, e.g., to get a model trained only on laion400m run:
```
open_clip.create_model_and_transforms('ViT-B-32-quickgelu', pretrained='laion400m_e32')
```
and to get the CLIP ViT-B/32 model which was trained on the OpenAI CLIP dataset run:
```
open_clip.create_model_and_transforms('ViT-B-32-quickgelu', pretrained='openai')
```
Check out the [colab](https://colab.research.google.com/github/mlfoundations/open_clip/blob/master/docs/Interacting_with_open_clip.ipynb) for more info. To see all options available you can run `open_clip.list_pretrained()`.

**Comment by rwightman** [comment]
@hawkfly07 moving this to discussions as it's not a bug or feature, and might be useful for others in the future

**Comment by MengSunS** [comment]
whats the difference between  ('ViT-B-32', 'openai') and ('ViT-B-32-quickgelu', 'openai')?

**Reply by mitchellnw** [reply]
see https://github.com/mlfoundations/open_clip/issues/441
